soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -3737,7 +3737,4 @@ void FileChoose_Init(GameState* thisx) {
     Font_LoadOrderedFont(&this->font);
     Audio_QueueSeqCmd(0xF << 28 | SEQ_PLAYER_BGM_MAIN << 24 | 0xA);
     func_800F5E18(SEQ_PLAYER_BGM_MAIN, NA_BGM_FILE_SELECT, 0, 7, 1);
-
-    // Originally this was only set when transitioning from the title screen, but gSkipLogoTitle skips that process so we're ensuring it's set here
-    gSaveContext.gameMode = GAMEMODE_FILE_SELECT;
```

Comment:
This was an old source change added because of the previous skip to file select enhancement. This can be dropped now that we have the improved boot sequence handling (which will guarantee the gamemode is set to `GAMEMODE_FILE_SELECT` before loading the file select state. (see my previous comment)